migrations/01-create-artist-table.sql

```diff
@@ -0,0 +1,5 @@
+ CREATE TABLE Artists (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  genre VARCHAR(255) NOT NULL
+  );
```

Comment:
The formatting is out here, remember to indent your code when writing inside brackets and have the closing bracket on the same level as the line that started it like this:
```
CREATE TABLE Artists (
  id SERIAL PRIMARY KEY,
  name VARCHAR(255) NOT NULL,
  genre VARCHAR(255) NOT NULL
);
```